File: WebService/App_Code/Models/TipoRelato.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

/// <summary>
/// Summary description for TipoRelato
/// </summary>
public class TipoRelato
{

    public int id_tipoRelato { get; set; }
    public string nome_tipoRelato { get; set; }


    public TipoRelato()
    {
    }

    public TipoRelato(int ID_TIPORELATO, string NOME_TIPORELATO)
    {
        this.id_tipoRelato = ID_TIPORELATO;
        this.nome_tipoRelato = NOME_TIPORELATO;
    }

}

File: Android/app/src/main/java/com/example/tcc_grupoc/TipoRelato.java
package com.example.tcc_grupoc;

public class TipoRelato {

    private int id_tipoRelato;
    private String nome_tipoRelato;

    public TipoRelato(int id_tipoRelato, String nome_tipoRelato)
    {
        this.id_tipoRelato = id_tipoRelato;
        this.nome_tipoRelato = nome_tipoRelato;
    }

    public int getId_tipoRelato() {
        return id_tipoRelato;
    }

    public String getNome_tipoRelato() {
        return nome_tipoRelato;
    }
}
